.platform/routes.yaml

```
@@ -168,6 +168,62 @@ https://{default}/:
       "/dedicated-gen-3/overview.html" : { "to": "/dedicated-gen-3.html", "code": 301 }
       "/dedicated-gen-2/overview/onboarding.html" : { "to": "/dedicated-gen-2.html", "code": 301 }
 
+      # Redirects for broken backlinks
+      "/administration/web/email.html" : { "to": "/development/email.html", "code": 301 }
+      "/administration/web/email.html#sending-e-mail][1]" : { "to": "/development/email.html", "code": 301 }
+      "/administration/web/email.html#sending-e-mail%5D%5B1%5D" : { "to": "/development/email.html", "code": 301 }
+      "/development/going-live.html" : { "to": "/domains/steps.html", "code": 301 }
+      "/development/going-live.html#prerequisites" : { "to": "/domains/steps.html", "code": 301 }
+      "/development/protective-block.html" : { "to": "/security.html", "code": 301 }
+      "/drupal_migrate/guides/local/git-initialize-repository.html" : { "to": "/tutorials/migrating.html", "code": 301 }
+      "/drupal_migrate/guides/type/php/drupal/migrate/import-database.html" : { "to": "/tutorials/migrating.html", "code": 301 }
+      "/drupal_migrate/guides/type/php/drupal/migrate/import-files.html" : { "to": "/tutorials/migrating.html", "code": 301 } 
+      "/drupal_migrate/guides/type/php/drupal/rebuild-site-registry.html" : { "to": "/tutorials/migrating.html", "code": 301 }
+      "/drupal/guides/local/get-project.html" : { "to": /guides/drupal9/deploy.html", "code": 301 } 
+      "/drupal/guides/prerequisites/platform-cli.html" : { "to": /guides/drupal9/deploy.html", "code": 301 } 
+      "/frameworks/drupal.html" : { "to": /guides/drupal9/deploy.html", "code": 301 } 
+      "/golive/steps/tls.html#optional-configure-a-third-party-tls-certificate" : { "to": "/domains/steps/tls.html", "code": 301 }
+      "/guides/general/composer-auth.html" : { "to": "/php/composer-auth.html", "code": 301 }
+      "/images/getting-started/free-trial.png" : { "to": "/get-started/introduction.html", "code": 301 }
+      "/images/integrations/bitbucket_server.png" : { "to": "/integrations/source/bitbucket.html", "code": 301 }
+      "/images/xdebug/xdebug-servers.png" : { "to": "/languages/php/xdebug.html", "code": 301 }
+      "/images/xdebug/xdebug-settings.png" : { "to": "/languages/php/xdebug.html", "code": 301 }
+      "/integration/blackfire.html" : { "to": "/increase-observability/integrate-observability/blackfire.html", "code": 301 }
+      "/integrations/observability/blackfire.html" : { "to": "/increase-observability/integrate-observability/blackfire.html", "code": 301 }
+      "/integrations/source/bitbuckethtml" : { "to": "/integrations/source/bitbucket.html", "code": 301 }
```

@AnouckColson was this line supposed to be this?

```suggestion
      "/integrations/source/bitbucket.html" : { "to": "/integrations/source/bitbucket.html", "code": 301 }
```